tools/src/main/scala/scala/scalanative/compiler/analysis/Shows.scala

```diff
@@ -0,0 +1,52 @@
+package scala.scalanative
+package compiler
+package analysis
+
+import ControlFlow.Block
+import nir.Shows._
+
+object Shows {
+
+  private def blockToString(block: Block): String =
+    showLocal(block.name).toString
+
+  def showCFG(cfg: ControlFlow.Graph): String = {
+    cfg.all.map { block =>
+      val succStr =
+        block.succ.map(_.to).map(blockToString).mkString("(", ",", ")")
```

Comment:
`block.succ.map(_.to)` can be simplified to `block.pred` after refactoring I've suggested earlier.